src/com/ensa/gi4/controller/GestionMaterielController.java

```diff
@@ -3,26 +3,503 @@
+						System.out.println("5-supprimer un materiel");
+						System.out.println("6-marquer un materiel comme non disponible");
+						System.out.println("7-allouer un materiel");
+						System.out.println("8-rendre un materiel");
+						System.out.println("9-afficher la liste de mes materiels");
+						System.out.println("10-afficher la liste des materiels alloue par chaque user");
+						System.out.println("11-logout");
+						System.out.println("0-si vous voulez sortir de l'application");
+						int operation = scanner.nextInt();
+						System.out.println();
+						switch (operation) {
+						case 1:
+							afficherMateriel();
+							System.out.println();
+							break;
+						case 2:
+							search();
+							System.out.println();
+							break;
+						case 3:
+							createMaterial();
+							break;
+						case 4:
+							modifierMateriel();
+							System.out.println();
+							break;
+						case 5:
+							removeMateriel();
+							System.out.println();
+							break;
+						case 6:
+							markeAsUnAvailable();
+							System.out.println();
+							break;
+						case 7:
+							alocateMateriel(user.getId());
+							System.out.println();
+							break;
+						case 8:
+							renderMateriel(user.getId());
+							System.out.println();
+							break;
+						case 9:
+							myMaterielList(userName);
+							System.out.println();
+							break;
+						case 10:
+							usersMateriels();
+							System.out.println();
+							break;
+						case 11:
+							isLogged = false;
+							break;
+						case 0:
+							sortirDeLApplication();
+							break;
+
+						default:
+							System.out.println("choix invalide");
+							break;
+						}
+					}
+				} else {
+					while (isLogged) {
+						System.out.println("1-afficher la liste de tous les materiels");
+						System.out.println("2-chercher un materiel");
+						System.out.println("3-allouer un materiel");
+						System.out.println("4-rendre un materiel");
+						System.out.println("5-afficher la liste de mes materiels");
+						System.out.println("6-logout");
+						System.out.println("0-si vous voulez sortir de l'application");
+						System.out.println();
+						int operation = scanner.nextInt();
+						System.out.println();
+
+						switch (operation) {
+						case 1:
+							afficherMateriel();
+							System.out.println();
+							break;
+						case 2:
+							search();
+							System.out.println();
+							break;
+						case 3:
+							alocateMateriel(user.getId());
+							System.out.println();
+							break;
+						case 4:
+							renderMateriel(user.getId());
+							System.out.println();
+							break;
+						case 5:
+							myMaterielList(userName);
+							System.out.println();
+							break;
+						case 6:
+							isLogged = false;
+							break;
+						case 7:
+							sortirDeLApplication();
+							break;
+						default:
+							System.out.println("choix invalid");
+							break;
+						}
+					}
+				}
+			}
+		} catch (EmptyResultDataAccessException e) {
```

Comment:
this exception comes from the dao layer, shouldn't be here

Reply:
d'accord